apps/dashboard/app/javascript/dynamic_forms.js

```diff
@@ -415,10 +415,10 @@ function updateVisibility(event, changeId) {
   const val = valueFromEvent(event);
   const id = event.target['id'];
   let changeElement = undefined;
+
   $(`#${changeId}`).parents().each(function(_i, parent) {
     if(parent.classList.contains('form-group')) {
       changeElement = $(parent);
-      return false;
```

Comment:
The problem here is that `text_field` is being passed a `label` option, which wraps the label/input HTML combination in its own `.form-group` class. So far simply searching for the top `.form-group` parent has been easier than over-riding the `wrapper_class` on the `text_field`. ... But this isn't my favorite option. I'd rather figure out how to ensure that each widget is only wrapped in one `.form-group` div.